modules/aioseop_sitemap.php

```diff
@@ -3822,7 +3822,7 @@ public function parse_content_for_images( $content, &$images ) {
 		 *
 		 * @return array
 		 */
-		public function get_tax_args( $page = 0 ) {
+		public function get_tax_args( $taxonomy, $page = 0 ) {
```

Comment:
Documentation needs to updated, and the `since` tag could also reflect a change to the param(s).